ignite/engine/engine.py

```diff
@@ -114,10 +117,15 @@ class Custom_Events(Enum):
 
             engine = Engine(process_function)
             engine.register_events(*Custom_Events)
```

Comment:
Maybe we can provide another example here with `event_to_attr`

Reply:
Provide example in docstring.